content/api/versions/v1.1/rest-sdk/push-admin.textile

```diff
@@ -0,0 +1,85 @@
+---
+title: Push Notifications - Admin
+meta_description: "Client Library SDK REST API Reference Push documentation."
+meta_keywords: "Ably, Ably REST, API Reference, REST SDK, REST interface, REST API, Push"
+section: api
+index: 80
+languages:
+  - javascript
+  - java
+  - ruby
+  - php
+  - android
+  - nodejs
+  - swift
+  - objc
+jump_to:
+  Push Admin API properties:
+    - deviceRegistrations#device-registrations
+    - channelSubscriptions#channel-subscriptions
+  Push Admin API methods:
+    - publish
+  Types:
+    - Related Types#related-types
+redirect_from:
```

Comment:
The redirect for v1.1 gives a 404. I think it has to contain the full version path to redirect correctly for older versions: `/api/rest-sdk/versions/v1.1/push`. I'd suggest maybe we just remove it and only have a redirect on the latest. WDYT?

Reply:
I think that makes sense!